### PR TITLE
Initial exercise

### PR DESCRIPTION
**Pull request by rlmark**
Adds first exercise, hello world.

### CODE REVIEW

**Comment by iHiD** [comment]
@ErikSchierboom will take a look at this properly, but it looks great to me at a quick glance. Nice work, @rlmark :)

File: config.json [hunk 1]
```diff
@@ -23,9 +23,52 @@
...
+      {
+        "slug": "hello-world",
+        "name": "Hello World",
+        "uuid": "151ebc38-659c-4752-b51c-cc7d1d7b1ef4",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 1,
+        "topics": [
+          "strings"
+        ]
```
Comment (ErikSchierboom) [inline config.json:36]: The `topics` key is not actually used anymore. We've deprecated it in v3 in favor of `practices` and `prerequisites`. So you could consider removing the field.

**Review by ErikSchierboom — APPROVED** [review]
Excellent work! One tiny nit, but feel free to merge.

**Comment by ErikSchierboom** [comment]
🥳

**Pull request merged**